container/go/cmd/puller/BUILD

```diff
@@ -11,25 +11,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# ###
+# Build file for new puller binary based on go-containerregistry backend.
```

Comment:
Usually we don't write this kind of comment. But it doesn't hurt to have it. No need to change.